flake8_async/__init__.py

```diff
@@ -38,7 +38,7 @@
 
 
 # CalVer: YY.month.patch, e.g. first release of July 2022 == "22.7.1"
-__version__ = "24.5.6"
+__version__ = "24.5.7"
```

Comment:
```suggestion
__version__ = "24.6.1"
```

since I've taken so long 😅